overlay/base/usr/lib/system/_os/cli/install.py

```
@@ -128,7 +127,7 @@ def install(  # noqa: PLR0917
         print("Boot partition must be specified")
         sys.exit(1)
 
-    setattr(ostree, "repo", f"/mnt{getattr(ostree, 'repo')}")
+    setattr(ostree, "repo", os.path.normpath(f"{sysroot}/{getattr(ostree, 'repo')}"))
```

Ruff (0.16.1)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)